packages/@okta/vuepress-theme-prose/layouts/RedesignHomePage.vue

```diff
@@ -0,0 +1,316 @@
+            },
+             {
+              name: 'Vue.js',
+              link: '/go',
+              image: '/img/homepage/stackselectors/vue.png'
+            },
+          ]
+        },
+          {
+          name: "Native app",
+          categoryLink: "/docs/guides/sign-into-mobile-app/android/before-you-begin/",
+          items: [
+            {
+              name: 'native React',
+              link: '/go',
+              image: '/img/homepage/stackselectors/react.png'
+            },
+             {
+              name: 'android',
+              link: '/go',
+              image: '/img/homepage/stackselectors/android.png'
+            },
+             {
+              name: 'IOS',
+              link: '/go',
+              image: '/img/homepage/stackselectors/ios.png'
+            },
+          ]
+        },
+        {
+          name: "API services",
+          categoryLink: "/docs/guides/protect-your-api/aspnet/before-you-begin/",
+          items: [
+            {
+              name: 'Go',
+              link: '/go',
+              image: '/img/homepage/stackselectors/go.png'
+            },
+             {
+              name: 'Java',
+              link: '/go',
+              image: '/img/homepage/stackselectors/java.png'
+            },
+             {
+              name: '.net',
+              link: '/go',
+              image: '/img/homepage/stackselectors/dotnet.png'
+            },
+             {
+              name: 'Node.js',
+              link: '/go',
+              image: '/img/homepage/stackselectors/nodejs.png'
+            },
+             {
+              name: 'PHP',
+              link: '/go',
+              image: '/img/homepage/stackselectors/nodejs.png'
+            }
+          ]
+        },
+      ],
+      assuranceData:[
+        {
+          description: "with HIPAA, FedRamp, SOC and more, we have you covered",
+          title: "Compliant out of the Box",
+          img: "/img/homepage/compliance-out-of-box.png"
+        },
+          {
+          description: "so your apps are working even when you’re not",
+          title: "99.99% Uptime",
+          img: "/img/homepage/uptime.png"
+        },
+          {
+          description: "SAML, OpenID Connect, and OAuth 2.0 connect you to everything",
+          title: "Open Standards",
+          img: "/img/homepage/standards.png"
+        }
+      ],
+      partnerData: [
+        {
+          alt: 'major league baseball',
+          src: '/img/homepage/partners/baseball.png'
+        },
+         {
+          alt: 'splunk',
+          src: '/img/homepage/partners/splunk.png'
+        },
+          {
+          alt: 'adobe',
+          src: '/img/homepage/partners/adobe.png'
+        },
+          {
+          alt: 'jet-blue',
+          src: '/img/homepage/partners/jet-blue.png'
+        },
+          {
+          alt: 'experian',
+          src: '/img/homepage/partners/experian.png'
+        },
+      ],
+    }
+  },
```

Comment:
maybe it makes sense to move constant data in a separate file. what do you think?

Reply:
We can use the yml format allowed inside the Frontmatter markdown files

Reply:
moved